nvtabular/loader/backend.py

```diff
@@ -142,9 +177,13 @@ def load_chunks(self, dev, dataloader):
                         spill = workflow.apply_ops(spill)
                     spill = dataloader._create_tensors(spill)
                     spill = dataloader._handle_tensors(*spill)
-                    self.put([spill])
+                    # ADDING FOR SYNC BEHAVIOR
+                    # self.put([spill])
+                    yield spill
```

Comment:
Should this be ```yield [spill]``` instead to match previous behaviour?

Reply:
ah good catch yes it should